MiBench/automotive/basicmath/src/basicmath_large.c

```diff
@@ -89,35 +99,42 @@ int main(void)
   }
 
 
+
   printf("********* INTEGER SQR ROOTS ***********\n");
-  /* perform some integer square roots */
-  for (i = 0; i < 100000; i+=2)
-    {
-      usqrt(i, &q);
-			// remainder differs on some machines
-     // printf("sqrt(%3d) = %2d, remainder = %2d\n",
-     printf("sqrt(%3d) = %2d\n",
-	     i, q.sqrt);
-    }
-  printf("\n");
+  
+  start = clock();
+  for (i = 0; i < 100000; i += 2)
+  {
+    usqrt(i, &q);
+    // remainder differs on some machines
+    printf("sqrt(%3d) = %2d\n", i, q.sqrt);
+  }
+  
+  end = clock();
+  cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;
+ 
+
+ printf("\n");
   for (l = 0x3fed0169L; l < 0x3fed4169L; l++)
     {
 	 usqrt(l, &q);
```

Comment:
Why is this loop not included in the timing measurement?

Reply:
For the first loop, I could understand the range. But not for the second one, so was confused. I am adding timers for the second loop too.